source/assets/sass/components/_playground.scss

```diff
@@ -0,0 +1,321 @@
+@use 'sass:color';
+@use '../breakpoints';
```

Comment:
Not specific to this branch, but consider moving `_breakpoints.scss` into the config folder - since it doesn't have css output of it's own? Same with `_functions.scss`?

Reply:
https://trello.com/c/V2ujZ5Pc/5-split-config-only-css-from-the-css-that-has-output Added this suggestion to the card checklist ... but I probably can quickly take care of this.

Reply:
done: https://github.com/oddbird/sass-site/pull/58/commits/31d9b56e1e9cc6e0ff132c926354dc7f6f82bb2e